app/services/cfe_service.rb

```diff
@@ -0,0 +1,156 @@
+class CfeService
+  def self.call(cfe_estimate_id, cfe_session_data)
+    new.call(cfe_estimate_id, cfe_session_data)
+  end
+
+  def call(cfe_estimate_id, cfe_session_data)
+    applicant_screen_create cfe_estimate_id, Flow::ApplicantHandler.model(cfe_session_data)
+    save_employment_data cfe_estimate_id, Flow::EmploymentHandler.model(cfe_session_data)
+    save_monthly_income_data cfe_estimate_id, Flow::MonthlyIncomeHandler.model(cfe_session_data)
+    save_vehicle_value_data cfe_estimate_id, Flow::Vehicle::ValueHandler.model(cfe_session_data)
+    save_vehicle_finance_data cfe_estimate_id, cfe_session_data
+    save_assets_data cfe_estimate_id, cfe_session_data
+    save_outgoings_data cfe_estimate_id, cfe_session_data
+    save_property_entry_data cfe_estimate_id, Flow::PropertyEntryHandler.model(cfe_session_data)
+    create_applicant cfe_estimate_id, Flow::ApplicantHandler.model(cfe_session_data)
+  end
+
+  def cfe_connection
```

Comment:
All these methods should be marked private as I don't think they are part of the public interface.